Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "edn-rs"
 version = "0.17.4"
-authors = ["Julia Naomi <jnboeira@outlook.com>",  "Eva Pace <eba.pachi@gmail.com>"]
+authors = ["Julia Naomi <jnboeira@outlook.com>",  "Eva Pace <eba.pachi@gmail.com>", "Kevin Nakamura <grinkers@grinkers.net>"]
```

Comment:
Add your name as maintainer on the main README?

Reply:
Sure. I matched the other temporry repo and removed the note about being close to stable. I'll re-add that note after I rework EdnError.